assets/stylesheets/main.css

```diff
@@ -212,6 +214,38 @@ a:hover#join {
   border-bottom: 1px solid aliceblue;
 }
 
+/*
+    =================================================
+    All sections
+    =================================================
+*/
+
+/* All sections EXCEPT first, b/c it uses padding instead. This approach
+is from https://css-tricks.com/hash-tag-links-padding/ */
+
```

Comment:
Adds a and extra 'chunk' to beginning of each section, which hides under the bottom of the previous section.  This allows the WCG header to sit on top of that area, preventing the active section to be visually obscured by the header.